prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml

```diff
@@ -23,7 +23,7 @@ presubmits:
       preset-service-account: "true"
     spec:
       containers:
-      - image: gcr.io/istio-testing/prowbazel:0.5.12
+      - image: gcr.io/istio-testing/prowbazel:0.5.13
```

Comment:
How long do these jobs take? Seems like we should use RBE to speed things up.

Reply:
Most presubmit jobs take ~30mins, presubmit-release and postsubmit take ~2h.

I was thinking about adding RBE to it yesterday, now that it's working again, but if you want to do it yourself and/or if there is some Prow dedicated RBE buildfarm already, then that's great!

Reply:
I filled an issue, so that this wouldn't get lost: #2013